CONTRIBUTING.md

```diff
@@ -131,7 +140,7 @@ The developer or developers working on said feature are responsible for keeping
 git checkout main
 git pull origin main # get the latest
 git checkout my-feature-branch
-git rebase my-feature-branch main
+git rebase main
 ```
 
```

Comment:
I would just suggest to add a note somewhere in the rebase section to make it clear not to make any changes to the files when resolving conflicts, unless the changes are directly related to the rebase.

Reply:
Is there a phrasing you think would be more useful than what we have here: https://github.com/dcgtc/dgrants/pull/167/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R146

It says:

> If conflicts are found, you will have to resolve them now. See here for more info on resolving merge conflicts when rebasing. After successfully rebasing, force push back to your feature branch, but never force push to main:

Reply:
hmm, yeah I guess I'll have to figure out better phrasing - this seems a bit confusing to me, since it seems like it's saying to resolve merge conflicts during rebase. I think it would be better if it was explicitly clear that you can't really resolve merge conflicts (things that break other parts of the code because of new commits) until after rebasing is done. 

Even in our Discord chat, the wording is more clear:

> I think a better phrasing would be "no changes outside the merge conflict"
> But fixing things within the conflict is ok

Reply:
@apbendi any thoughts here?

Reply:
Added the following:

> In some circumstances, the process of rebasing and resolving merge conflicts may have broken the feature or functionality you were working on. This tends to occur when large changes were made on `main` to the same section of code being modified in a feature branch. In these cases, simply add additional commit(s) to the head of your feature branch correcting the problems.

Going to merge this PR, but if we feel there is still room for improvement, feel free to open an issue or correcting PR @mds1 @mk1r